src/controllers/poll.js

```diff
@@ -48,19 +48,51 @@ export const getPoll = async (req, res, next) => {
   try {
     const { id } = req.params;
     const poll = await Poll.findById(id);
+    if (!poll)
+      return res.status(404).json({ message: 'There is no poll with this ID' });
     return res.status(200).json({ poll });
   } catch (error) {
     res.status(404).json({ message: error.message });
     next(error);
   }
 };
 
+export const deletePoll = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const poll = await Poll.findByIdAndRemove(id);
```

Comment:
What does this return?

Reply:
This will return any poll.id match the passed  id from the request params and remove it

Reply:
Ah fair